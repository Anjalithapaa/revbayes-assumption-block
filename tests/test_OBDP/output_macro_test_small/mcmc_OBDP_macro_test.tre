#NEXUS

	Begin taxa;
	Dimensions ntax=10;
	Taxlabels
		f0
		f1
		f2
		f3
		f4
		f50
		f51
		f52
		f53
		f54
		;
End;

Begin trees;
tree TREE1 = [&R]((((((f0[&index=8]:1.041267,f4[&index=7]:1.041267)[&index=11,posterior=1.000000,age_95%_HPD={1.04127,1.04127}]:3.185344,f53[&index=6,age_95%_HPD={4.2,4.2}]:0.026610)[&index=12,posterior=1.000000,age_95%_HPD={4.22661,4.22661}]:0.170072,f50[&index=5,age_95%_HPD={3.114,3.114}]:1.282683)[&index=13,posterior=1.000000,age_95%_HPD={4.39668,4.39668}]:0.061418,f1[&index=9]:4.458101)[&index=14,posterior=1.000000,age_95%_HPD={4.4581,4.4581}]:0.704458,(((f2[&index=2]:3.398372,f3[&index=1]:3.398372)[&index=15,posterior=1.000000,age_95%_HPD={3.39837,3.39837}]:0.968353,f51[&index=3,age_95%_HPD={3.803,3.803}]:0.563724)[&index=16,posterior=1.000000,age_95%_HPD={4.36672,4.36672}]:0.585279,f52[&index=4,age_95%_HPD={3.948,3.948}]:1.004003)[&index=17,posterior=1.000000,age_95%_HPD={4.952,4.952}]:0.210555)[&index=18,posterior=1.000000,age_95%_HPD={5.16256,5.16256}]:0.130834,f54[&index=10,age_95%_HPD={5.161,5.161}]:0.132393)[&index=19,posterior=1.000000,age_95%_HPD={5.29339,5.29339}]:0.000000;
End;
