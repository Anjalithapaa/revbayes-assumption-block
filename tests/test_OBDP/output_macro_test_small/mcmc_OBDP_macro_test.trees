Iteration	Posterior	Likelihood	Prior	obd_tree
0	-73523.87	-73444.79	-79.08119	(f54[&index=10]:0.132393,((f1[&index=9]:4.458101,(((f0[&index=8]:1.041267,f4[&index=7]:1.041267)[&index=11]:3.185344,f53[&index=6]:0.026610)[&index=12]:0.170072,f50[&index=5]:1.282683)[&index=13]:0.061418)[&index=14]:0.704458,(f52[&index=4]:1.004003,(f51[&index=3]:0.563724,(f2[&index=2]:3.398371,f3[&index=1]:3.398371)[&index=15]:0.968353)[&index=16]:0.585279)[&index=17]:0.210555)[&index=18]:0.130834)[&index=19]:0.000000;
1	-73523.87	-73444.79	-79.08119	(f54[&index=10]:0.132393,((f1[&index=9]:4.458101,(((f0[&index=8]:1.041267,f4[&index=7]:1.041267)[&index=11]:3.185344,f53[&index=6]:0.026610)[&index=12]:0.170072,f50[&index=5]:1.282683)[&index=13]:0.061418)[&index=14]:0.704458,(f52[&index=4]:1.004003,(f51[&index=3]:0.563724,(f2[&index=2]:3.398371,f3[&index=1]:3.398371)[&index=15]:0.968353)[&index=16]:0.585279)[&index=17]:0.210555)[&index=18]:0.130834)[&index=19]:0.000000;
