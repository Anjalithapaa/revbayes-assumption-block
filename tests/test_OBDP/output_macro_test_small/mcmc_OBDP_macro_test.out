Iteration	alpha_morpho	alpha_seq	branch_rates	clock_morpho	er	lambda	mu	obd_tree	omega	psi	sf	start_time
0	1.001199	0.4313493	0.002713486	2.88891	[0.128632,0.305192,0.128118,0.166692,0.220642,0.0507245]	0.007298359	0.011636	(f54[&index=10]:0.132393,((f1[&index=9]:4.458101,(((f0[&index=8]:1.041267,f4[&index=7]:1.041267)[&index=11]:3.185344,f53[&index=6]:0.026610)[&index=12]:0.170072,f50[&index=5]:1.282683)[&index=13]:0.061418)[&index=14]:0.704458,(f52[&index=4]:1.004003,(f51[&index=3]:0.563724,(f2[&index=2]:3.398371,f3[&index=1]:3.398371)[&index=15]:0.968353)[&index=16]:0.585279)[&index=17]:0.210555)[&index=18]:0.130834)[&index=19]:0.000000;	0.406959	0.1150825	[0.515133,0.331686,0.0544707,0.0987111]	8.490851
1	1.001199	0.4313493	0.002713486	2.88891	[0.128632,0.305192,0.128118,0.166692,0.220642,0.0507245]	0.007298359	0.011636	(f54[&index=10]:0.132393,((f1[&index=9]:4.458101,(((f0[&index=8]:1.041267,f4[&index=7]:1.041267)[&index=11]:3.185344,f53[&index=6]:0.026610)[&index=12]:0.170072,f50[&index=5]:1.282683)[&index=13]:0.061418)[&index=14]:0.704458,(f52[&index=4]:1.004003,(f51[&index=3]:0.563724,(f2[&index=2]:3.398371,f3[&index=1]:3.398371)[&index=15]:0.968353)[&index=16]:0.585279)[&index=17]:0.210555)[&index=18]:0.130834)[&index=19]:0.000000;	0.406959	0.1150825	[0.515133,0.331686,0.0544707,0.0987111]	8.490851
